### PR TITLE
Add /opt/python3.5/bin to $PATH

### PR DESCRIPTION
**Pull request by duggelz**
Implements #45

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,6 +23,9 @@ RUN pip install --upgrade pip virtualenv
 # Install the Google-built interpreters
 ADD interpreters.tar.gz /
 
+# Add Google-built interpreters to the path
+ENV PATH $PATH:/opt/python3.5/bin
```
Comment (jonparrott) [inline Dockerfile]: Why put it at the end? It's it typical to put it at the start?

Comment (duggelz) [inline Dockerfile]: Paranoia from bitter experience.  E.g. somehow a file named "sh" or "true" ends up in /opt/python3.5/bin and everything goes crazy.  I always add things to the end of $PATH.

Tangential rant, this is why Python's handling of $PYTHONPATH is so nutty.  What, you have a directory named "os" in the current working directory?  Let me helpfully break everything for you.

Comment (jonparrott) [inline Dockerfile]: Heh, in this case we built with `altinstall` so it should be safe to put this at the start of the path.

**Comment by jonparrott** [comment]
Cool, agree about the debian case. We want ours to always be preferred.

File: Dockerfile (hunk 1 above)
Comment (duggelz) [inline Dockerfile]: $PYTHONPATH literally just bit me, I was running in a directory that had a "calendar" subdirectory.  Grr.

**Pull request merged**